### PR TITLE
[doc] Add L3 drop counters SNMP design

### PR DESCRIPTION
**Pull request by mykolaf**
Signed-off-by: Mykola Faryma <mykolaf@mellanox.com>

### CODE REVIEW

File: doc/Port_illegal_packets_drop_design.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+| ifOutQLen          | SAI_PORT_STAT_IF_OUT_QLEN           | |
+
+## Interface  types and IANA iftypes
+
+The `ifType` for each of the interfaces: 
+
+| Interface  | IANA Iftype | Already implemented?  |
+| ---------- |:--------------:| :---------:|
+| port       | 6 ethernetCsmacd       | +  |
+| RIF        | - | Need to add to port counters  |
```
Comment (qiluo-msft) [inline doc/Port_illegal_packets_drop_design.md]: >Need to add to port counters [](start = 19, length = 28)

The words are confusing in the column "Already implemented?". Maybe you could add one more column "Proposed implementation".

File: doc/Port_illegal_packets_drop_design.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
+# Port illegal packets drop design
+
+### High level design document
+
+# List of Tables
+* [Table 1: Revision](#revision)
+* [Table 2: Abbreviations](#definitionsabbreviation)
+* [Table 3: SAI cointer mapping](#rfc1213-etries-and-sai-counter-mapping)
```
Comment (qiluo-msft) [inline doc/Port_illegal_packets_drop_design.md:8]:
```suggestion
* [Table 3: SAI counter mapping](#rfc1213-etries-and-sai-counter-mapping)
```

Comment (qiluo-msft) [reply doc/Port_illegal_packets_drop_design.md:8]: @mykolaf Could you please check?

**Comment by liat-grozovik** [comment]
@lguohan as implementation is already in review, can you please approve so we can merge?

**Comment by mykolaf** [comment]
> @lguohan as implementation is already in review, can you please approve so we can merge?

Implementation is merged already.

**Pull request merged**